### ISSUE TITLE
Quantum engine job URLs

### ISSUE THREAD

**Issue by viathor** [quantumlib/Cirq#3305]
**Is your feature request related to a use case or problem? Please describe.**

URLs to the webpage showing jobs scheduled on the quantum engine requires knowing details of the structure of the URLs that users are unlikely to remember. Context: #3304.

**Describe the solution you'd like**

Add an easily discoverable function that returns the URL for a given quantum engine job, i.e. URL like

`https://console.cloud.google.com/quantum/programs/PROGRAM_ID/jobs/JOB_ID/overview?project=PROJECT_ID`

**What is the urgency from your perspective for this issue? Is it blocking important work?**

P3 - I'm not really blocked by it, it is an idea I'd like to discuss / suggestion based on principle

**Comment by wcourtney** [comment]
Great suggestion, thanks. What do you think of adding a "consoleUrl" method to each of the Engine resources? Specifically, EngineProgram, EngineJob, and EngineProcessor?

**Comment by 95-martin-orion** [comment]
Hey Will, this came up at Cirq sync. Is this still on the table?

**Comment by wcourtney** [comment]
Still on the table. It's filed under "usability" and "console transitions" for the QCS roadmap. This should be pretty simple though and probably doesn't need to wait. Assigning to Cheng as the new owner of `cirq_google`.

**Comment by dstrain115** [comment]
I don't think that this needs to be done as part of Cirq 1.0.  Adding console_url() is extra functionality.  Even if we "break backwards compatibility" by changing job URLs, we do not have public access to the API, so I don't think this is a crucial breakage (or even covered by our library guarantees, as it is external functionality).  I propose this should be after 1.0.

@95-martin-orion Do you agree?  If so, can you change this to after/1.0?

**Comment by dstrain115** [comment]
@wcourtney Do you think we should still do this, or should we close this issue?

**Comment by mhucka** [comment]
Closing this as there is it hasn't had activity in years, it doesn't seem to block anyone, and we have to triage more aggressively. If this action is in error or something changes in the future, someone can reopen this.
